app/analysis_options.yaml

```diff
@@ -43,7 +41,8 @@ linter:
     - package_prefixed_library_names
 #    - parameter_assignments
     - prefer_final_fields
-    - prefer_final_locals
+# TODO: enable after generated code passes
```

Comment:
Is a bug filed? How about the other commented out elements here?

Reply:
https://github.com/dart-lang/json_serializable/issues/290

I think we haven't really revised the other elements in ages, it would be worth to take a fresh look (e.g. check if new rules could be applied).